modules/RTC/JitsiLocalTrack.js

```diff
@@ -559,13 +559,18 @@ export default class JitsiLocalTrack extends JitsiTrack {
      * NOTE: used only for audio tracks to detect audio issues.
      */
     _setByteSent(tpc, bytesSent) {
-        this._bytesSent = bytesSent;
+        this._bytesSentPerPC.set(tpc.id, bytesSent);
```

Comment:
since we end up adding those values anyway during the check, is there any point in keep a map instead of just a singe var as an accumulator ?

Reply:
@paweldomas See the comment above.